projects/ad7616-st/README.rst

```diff
@@ -0,0 +1,8 @@
+Evaluating AD7616 on SDP-K1
+===========================
+
+To setup and test AD7616 on the SDP-K1, please refer to the `evaluation guide <https://wiki.analog.com/resources/eval/user-guides/ad7616-sdz#sdp-k1_setup>`_
+
+The project provides an iio device over the serial interface. The baudrate used
+is 230400. You can use the `IIO Oscilloscope <https://wiki.analog.com/resources/tools-software/linux-software/iio_oscilloscope>`_
+to view the captured data.
```

Comment:
The content from the wiki page (related to this project) should be ported in the .rst file, except the images, but I think the information is quite clear even without the setup image.
Also, if possible, follow the documentation template from here: https://github.com/analogdevicesinc/no-OS/tree/main/projects/eval-adis1650x